app/src/main/java/org/mozilla/tv/firefox/onboarding/OnboardingActivity.kt

```diff
@@ -34,10 +34,7 @@ class OnboardingActivity : AppCompatActivity() {
     }
 
     private fun setTurboMode(turboModeEnabled: Boolean) {
-        PreferenceManager.getDefaultSharedPreferences(this)
-                .edit()
-                .putBoolean(Settings.TRACKING_PROTECTION_ENABLED_PREF, turboModeEnabled)
-                .apply()
+        TurboMode.toggle(this, turboModeEnabled)
```

Comment:
@dnarcese

Reply:
Nit: can you change this from `toggle` to `setEnabled`? toggle isn't entirely clear.